docs/node-mixin/alerts/alerts.libsonnet

```diff
@@ -248,6 +248,33 @@
               message: 'Clock on {{ $labels.instance }} is not synchronising. Ensure NTP is configured on this host.',
             },
           },
+          {
+            alert: 'NodeRAIDDegraded',
+            expr: |||
+              'node_md_disks_required - ignoring (state) (node_md_disks{state="active"}) != 0'
+            ||| % $._config,
+            'for': '15m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              summary: 'RAID Array is degraded',
+              message: 'RAID array '{{ $labels.device }}' is in degraded state due to one or more disks failures. Number of spare drives is insufficient to fix issue automatically.',
+            },
+          },
+          {
+            alert: 'NodeRAIDDiskFailure',
+            expr: |||
+              node_md_disks{state="fail"} > 0
+            ||| % $._config,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'Failed device in RAID array on {{ $labels.instance }}',
```

Comment:
I'd prefer the summary to be static (or at least to render into the same string for typical alert groupings). That's consistent with the other usage in this file. See also https://github.com/prometheus/node_exporter/issues/1765#issuecomment-668653794

Reply:
fixed.